packages/devtools_app/lib/src/shared/utils.dart

```diff
@@ -44,16 +42,15 @@ void debugLogger(String message) {
 }
 
 double scaleByFontFactor(double original) {
-  return (original * (ideTheme?.fontSizeFactor ?? 1.0)).roundToDouble();
+  return (original * (ideTheme.fontSizeFactor ?? 1.0)).roundToDouble();
 }
 
 bool isDense() {
-  return preferences != null && preferences.denseModeEnabled.value ||
-      isEmbedded();
+  return preferences.denseModeEnabled.value || isEmbedded();
 }
 
 bool isEmbedded() {
-  return ideTheme?.embed ?? false;
+  return ideTheme.embed ?? false;
```

Comment:
make embed non-nullable also. you can give it a default value of false in the constructor

Reply:
done